### PR TITLE
rephrase, small corrections

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
made small edits to outline.md to include spelling correction, rephrasing, link to SPIFFE API doc

### CODE REVIEW

File: assessments/projects/spiffe-spire/outline.md [hunk 1]
```diff
@@ -14,30 +14,30 @@ Project team: Andres Vega, Andrew Jessup, Tyler Julian, Andrew Harding.
 
 ## Background
 
-SPIFFE and SPIRE are a set of APIs and associated tooling that provides a uniform language for describing service identity in a wide range of workloads (including orchestration systems), verifying that identity, and providing a workload with documents that serve as proof of that identity. 
+Secure Protection Identity Framework for Everyone (SPIFFE) are a set of APIs with SPIRe as the production grade implemtnation and associated tooling that provide a uniform language for describing service identity in a wide range of workloads (including orchestration systems), verifying that identity, and providing a workload with documents that serve as proof of that identity.
```
Comment (lumjjb) [inline assessments/projects/spiffe-spire/outline.md]: nit: 
SPIRE
implemtnation typo

"the production grade" - is the project is making this statement?

File: assessments/projects/spiffe-spire/outline.md [hunk 2]
```diff
@@ -14,30 +14,30 @@ Project team: Andres Vega, Andrew Jessup, Tyler Julian, Andrew Harding.
...
+Secure Protection Identity Framework for Everyone (SPIFFE) are a set of APIs with SPIRe as the production grade implemtnation and associated tooling that provide a uniform language for describing service identity in a wide range of workloads (including orchestration systems), verifying that identity, and providing a workload with documents that serve as proof of that identity. 
 
 _Maturity_
 
 (Examples)
-- Known end-users include: Bloomberg, Bytedance, Pinterest, Square, Uber, and Yahoo Japan. This is not an exclusive list and it’s likely that due to the secrecy of a security framework other organizations are also SPIFFE and SPIRE users privately.
+- Known end-users include: Bloomberg, Bytedance, Pinterest, Square, Uber, and Yahoo Japan. This is not an exhaustive list, due to the sensitivity surrounding exposure of security frameworks in use it is likely other organizations are also SPIFFE and SPIRE users privately.
```
Comment (lumjjb) [inline assessments/projects/spiffe-spire/outline.md]: hmm maybe rephrase this to

"This is not an exhaustive list as other organizations may use SPIFFE and SPIRE privately (due to the sensitivity surrounding exposure of security frameworks being used)"

File: assessments/projects/spiffe-spire/outline.md [hunk 3]
```diff
@@ -14,30 +14,30 @@ Project team: Andres Vega, Andrew Jessup, Tyler Julian, Andrew Harding.
...
-The project fulfills the role of a layer below the orchestration framework, and for its effectiveness, consumers of identitiy bundles need to have a good understanding of the demarcation line of security responsibilities.
+The project fulfills a needed role layer below the orchestration framework layer, and for its effectiveness, consumers of identitiy bundles need to have a good understanding of the demarcation line of security responsibilities.
 
 All questions from reviewers were addressed in [self-assessment](self-assessment.md)
 with non-critical issues captured as issues and noted below.
 
 ## Recommendations
 
-### Recommendations to the project team
+### Recommendations to the Project Team
```
Comment (lumjjb) [inline assessments/projects/spiffe-spire/outline.md]: https://github.com/cncf/sig-security/blob/master/CONTRIBUTING.md

This should follow sentence case

File: assessments/projects/spiffe-spire/outline.md [hunk 4]
```diff
@@ -14,30 +14,30 @@ Project team: Andres Vega, Andrew Jessup, Tyler Julian, Andrew Harding.
...
+**Design**: Covering many use cases of identity without exposing unnecesary complexity to the user, SPIFFE/SPIRE provide a streamlined and simple design to interact with. The SPIFFE '[Workload API](https://spiffe.io/spiffe/concepts/#spiffe-workload-api)' provides a great way for services and orchestration systems to integrate. This pluggable design for attestation, key management, databases, and more allows for easy extensibility.
 
 **Analysis**: The project, as a security provider, has done due diligence in security and threat modeling. The security workflow is evident, and the project is in the correct direction to further improve its security process and CI verification. 
-The project fulfills the role of a layer below the orchestration framework, and for its effectiveness, consumers of identitiy bundles need to have a good understanding of the demarcation line of security responsibilities.
+The project fulfills a needed role layer below the orchestration framework layer, and for its effectiveness, consumers of identitiy bundles need to have a good understanding of the demarcation line of security responsibilities.
```
Comment (lumjjb) [inline assessments/projects/spiffe-spire/outline.md]: I don't know if we want to say it's a "needed" role. The design decision of making this a layer below orchestration is a choice that the project made. I think we don't want to be prescriptive that this layer is mandated and there could be trade-offs with doing this as part as orchestration.

File: assessments/projects/spiffe-spire/outline.md (hunk 1 above)
Comment (TheFoxAtWork) [reply assessments/projects/spiffe-spire/outline.md]: corrected, changed to production-ready

File: assessments/projects/spiffe-spire/outline.md (hunk 3 above)
Comment (TheFoxAtWork) [reply assessments/projects/spiffe-spire/outline.md]: corrected

File: assessments/projects/spiffe-spire/outline.md (hunk 4 above)
Comment (TheFoxAtWork) [reply assessments/projects/spiffe-spire/outline.md]: concur, corrected

File: assessments/projects/spiffe-spire/outline.md (hunk 2 above)
Comment (TheFoxAtWork) [reply assessments/projects/spiffe-spire/outline.md]: love that wording better!
